src/sagemaker/image_uri_config/mxnet.json

```diff
@@ -408,14 +408,17 @@
                     "ap-northeast-1": "520713654638",
                     "ap-northeast-2": "520713654638",
                     "ap-south-1": "520713654638",
+                    "ap-south-2": "772153158452",
```

Comment:
Please remove these entries for all framework versions whose repository is `sagemaker-mxnet`, `sagemaker-mxnet-serving`, `sagemaker-mxnet-eia`, and `sagemaker-mxnet-serving-eia`.